OptimizelySDK/Odp/OdpEventManager.cs

```diff
@@ -349,7 +349,7 @@ public void IdentifyUser(string userId)
         /// <param name="odpConfig">Configuration object containing new values</param>
         public void UpdateSettings(OdpConfig odpConfig)
         {
-            _odpConfig.Update(odpConfig.ApiKey, odpConfig.ApiHost, odpConfig.SegmentsToCheck);
+            _odpConfig = odpConfig;
```

Comment:
If you use to replace _odpConfig, don't we need to set it to "volitile"?

Reply:
`volatile` can [only be set on some simple/primitive types](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/volatile).  

Replacing the config as a whole now (instead of individual field values on the OdpConfig) should not be good. I'm not seeing where a child thread updates on the config.